### PR TITLE
added all files for emptyquizresults_page

### LINKED ISSUE #24: Storybook: Empty Quiz Result Page Component
## User story

**Who:** **As a** developer
**What:** **I want to** be able to redirect the users of the app when the the answers don't give a result
**Why:** **so that I can** inform the users that their Quiz didn't return a result due to the lack of answers/skipped answers.

## Acceptance criteria

Create a storybook component for the Empty Quiz Result Page like shown below:

Use the MENU - https://github.com/HackYourFuture-CPH/Guide-IT/issues/14 
Use the Button - https://github.com/HackYourFuture-CPH/Guide-IT/issues/7
And the images reffered in the end of this story

Screenshot 2021-04-13 at 19 50 12

Check the wireframes here: https://www.figma.com/file/9SJRaxPSuPI3lSl8ULNyJ8/GuideIT-For-Developers?node-id=0%3A1

## Implementation details

Use the image **Vector.svg** located in this folder **/src/client/assets/images/** as a **background image** and use the image below girl.png on the page, this image should be added to the same folder

girl

And you should also use the **Robot image** that should be already in the same image folder.

### PR DESCRIPTION
**Pull request by rajanibusani**
# Description
Created Empty Quiz Result Page Component 

Please provide a short summary explaining what this PR is about.

Fixes #24  (issue)

# How to test?
npm run storybook
Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.component.js [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+            </div>
+          </div>
+          <Buttons label="Go Home" size="big" isMono={true} onClick={onClick} />
+        </div>
+      </div>
+    </section>
+  );
+};
+
+// const EmptyQuizResult = () => {
```
Comment (sofiiadidovych) [inline src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.component.js:37]: Rajani, I think that this commented code can be deleted as it isn`t used.

**Review by sofiiadidovych — APPROVED** [review]
Good job, Rajani! Everything looks good to me, the code is clear.

**Review by sandhana14 — APPROVED** [review]
Everything looks great

**Comment by ccshah298** [comment]
good job @rajanibusani ..all components looks nice.. 👍

File: src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.component.js [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+//         <h2 className="noresults_text">No results</h2>
+//         <img src={girlImg} className="girl_img" alt="girlplaying"></img>
+//         <div className="robo_container">
+//           <div className="sadrobo">
+//             <img src={sadrobo} alt="disappointed robo face"></img>
+//           </div>
+//           <div className="sadrobo_text">
+//             <p>You've skipped too many questions</p>
+//           </div>
+//         </div>
```
Comment (bytewiz) [inline src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.component.js:54]: remove commented code 👍

**Review by bytewiz — CHANGES_REQUESTED** [review]
Remove the commented code then I can approve it 👍

File: src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.component.js (hunk 2 above)
Comment (rajanibusani) [reply src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.component.js]: @bytewiz , i deleted that commented code and renamed my components...thank you

**Review by bytewiz — APPROVED** [review]

**Pull request merged**